packages/client/src/features/accessibility/element-navigation/local-element-navigator.ts

```diff
@@ -0,0 +1,147 @@
+/********************************************************************************
+ * Copyright (c) 2023 Business Informatics Group (TU Wien) and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+import { BoundsAware, EdgeRouterRegistry, SConnectableElement, SEdge, SModelElement, SModelRoot, TYPES } from '~glsp-sprotty';
+import { ElementNavigator } from './element-navigator';
+import { inject, injectable, optional } from 'inversify';
+import { GLSPActionDispatcher } from '../../../base/action-dispatcher';
+import { applyCssClasses, deleteCssClasses } from '../../../base/feedback/css-feedback';
+import { SelectableBoundsAware } from '../../../utils/smodel-util';
+
+@injectable()
+export class LocalElementNavigator implements ElementNavigator {
+    navigableElementCSS = 'navigable-element';
```

Comment:
It seems that this CSS class is not properly applied to the element currently focused in the navigation mode. Could you please doublecheck why they are not properly applied? Thanks!